.github/workflows/docker-test-containers-daily.yml

```diff
@@ -5,8 +5,14 @@ on:
     - cron: "23 3 * * *"
   workflow_dispatch:
 
+permissions:
+  contents: read
+
 jobs:
   copy-images:
+    permissions:
+      contents: read
```

Comment:
Is this redundant?
```suggestion
```

Reply:
https://github.com/open-telemetry/opentelemetry-java/pull/7070#discussion_r1945611642

Reply:
these are needed after all: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13247